app/assets/images/sephora_style_guide/svgs/flags/australia.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="30px" height="20px" viewBox="0 0 30 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Might wanna compress the file. We can remove `width` and `height` and use only `viewBox`. Also, we can do without the `xml` tag above and the generator comment below.